lib/environment_light.rb

```diff
@@ -3,7 +3,7 @@
 # Set up Rails configuration
 Rails = Struct.new(:env,:root,:logger).new
 Rails.env  = ENV['RAILS_ENV'] || 'production'
-Rails.root = Pathname.new(File.dirname(__FILE__) << "/..")
+Rails.root = Pathname.new(File.dirname(__FILE__) << '/..')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.